osf/models/provider.py

```diff
@@ -83,7 +87,6 @@ def get_default(cls):
     )
     share_source = models.CharField(blank=True, default='', max_length=200)
     share_title = models.TextField(default='', blank=True)
-    access_token = EncryptedTextField(null=True, blank=True)
```

Comment:
Why should the access token field be removed?

Reply:
yeah sorry, I should have left a note here, but look down the page, it's a duplicate instantiation of access_token, and doesn't do anything.